lib/staging/tls/tls.hpp

```diff
@@ -57,6 +59,28 @@ class ConfigItem {
     }
 };
 
+class TlsKeyLoggingServer {
+public:
+    TlsKeyLoggingServer(const std::string& interface_name, uint16_t port_);
+    ~TlsKeyLoggingServer();
+
+    ssize_t send(const char* line);
+
+    auto get_fd() const {
+        return fd;
+    }
+
+    auto get_port() const {
+        return port;
+    }
+
+private:
+    int fd{-1};
+    std::array<uint8_t, 2048> buffer{};
```

Comment:
I'm not sure buffer is used any more.

Reply:
You are right, buffer is not used. I will delete this one